runtime/doc/starting.txt

```diff
@@ -920,7 +920,7 @@ The ShaDa file is used to store:
 - Marks for several files.
 - File marks, pointing to locations in files.
 - Last search/substitute pattern (for 'n' and '&').
-- The buffer list.
+- The buffer list (without |nobuflisted| buffers).
```

Comment:
`|nobuflisted|` is not a valid tag, but anyway let's change this to:
```suggestion
- The buffer list (only 'buflisted' buffers).
```

Reply:
Thanks. Makes sense to remove the double negation